### DISCUSSION TITLE
Air sdk 33.1.1.713 non-security server warning android apps below android 9

### DISCUSSION THREAD

**Discussion by mehmury** [airsdk/Adobe-Runtime-Support#1475] [Q&A]
I tested my android applications with Air sdk version 33.1.1.713. In my applications, I receive data over my domain connections with https.
However, after the application is opened, when URL Request runs, Adobe air shows a warning. It reports that the domain I am using is not secure. This pop-up warning appears in from below android 9 
![Screenshot_2021-12-18-09-47-57](https://user-images.githubusercontent.com/51246471/146632351-2979c58e-fd42-43d2-9741-73009dece2b1.png)
 versions. This problem does not exist in higher versions.
What is the cause of this warning? How can I fix it?
thanks

**Comment by leossmith** [comment]
That is a typical issue with air's or servers certificate expiring. I had to pass everything through cdn in an app to avoid that earlier this year.

**Comment by FliplineStudios** [comment]
@mehmury Any idea if this is related to #1228 from when Let's Encrypt's old root certificate expired at the end of September?  I believe some older iOS and Android devices may not trust the newer root/intermediate certificate that Let's Encrypt uses now, so it's flagged as not being secure.  We also had to switch to a paid SSL cert that devices would trust to get around the problem.

**Reply by mehmury** [reply]
I switched to a paid ssl certificate. However, I still keep getting the same warning. I think there is a problem with the certificate of Air sdk.
![Screenshot_2021-12-19-11-58-39](https://user-images.githubusercontent.com/51246471/146669369-51ba9735-6bc1-4176-9f00-35c101caa61e.png)

**Comment by mehmury** [comment]
The ssl certificate I use on the server seems valid and without any problems. For air 33.1.1.713 sdk, I still get these warnings in Android old versions. I will wait for the new air sdk version.
If the warning still persists, it looks like it might be a problem in my future app updates.
![ssll](https://user-images.githubusercontent.com/51246471/146676009-03c8a865-8fc6-4f26-bf01-0055792294d3.png)

**Reply by ajwfrost** [reply]
Hi

In AIR on Windows, I get:
![image](https://user-images.githubusercontent.com/6400804/146747605-be39e667-955b-40c8-9186-82a7d628dde8.png)

Which basically means we can't check your certificate issuer's revocation list.. because the URL is missing from the certificate. So e.g. your certificate details have:
![image](https://user-images.githubusercontent.com/6400804/146748838-c01bf859-a827-4741-9ca9-32f6dc83c8ec.png)
whereas another website (ssl247.co.uk) has:
![image](https://user-images.githubusercontent.com/6400804/146748917-ed667859-4e40-4389-b40b-9c75f4661895.png)

It's curious that it works in newer Android versions, but it's possible that they have some local mechanism for checking relevant revocation lists?

**Reply by raresn** [reply]
I get this for a few of my clients as well.
The only way i could solve it is by removing the check for SSL from internet explorer-> internet settings

**Reply by mehmury** [reply]
I bought the ssl certificate for my domain 1 day ago. Could it be because it was just added that it is blank in the list? Can it be added after a while?

**Comment by mehmury** [comment]
I solved the cause of the problem. I checked the devices I tested. My android devices that gave this warning were registered to the company account on Google Works. Such a mistake occurred because they were old android versions.
I removed one of the devices from my Google Works account and the warning went away. It appears that there is a valid security warning for devices registered with Google Works.

Problem solved.
Thanks

**Reply by ajwfrost** [reply]
Okay interesting! I would still have said that the certificate is missing a field (I think this is just part of the certificate generation options rather than being anything caused by the fact it was only recently generated etc..) -> but at least it seems Android is consistent in how it's handled! 

thanks

**Comment by walddada** [comment]
hello
I have a problem I don't know is for me or not
when I publish APK from Adobe Animate cc 
The APK file doesn't work in my emulator

adobe sdk sdk_33_1_1_680

thanks
